sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_autoinstrumentation/configurator.py

```diff
@@ -16,25 +15,35 @@
 )
 from azure.monitor.opentelemetry._diagnostics.diagnostic_logging import (
     AzureDiagnosticLogging,
+    _ATTACH_FAILURE_CONFIGURATOR,
+    _ATTACH_SUCCESS_CONFIGURATOR,
+)
+from azure.monitor.opentelemetry._diagnostics.status_logger import (
+    AzureStatusLogger,
 )
-
-_logger = logging.getLogger(__name__)
 
 
 class AzureMonitorConfigurator(_OTelSDKConfigurator):
     def _configure(self, **kwargs):
         if not _is_attach_enabled():
             warn(_PREVIEW_ENTRY_POINT_WARNING)
         try:
-            AzureDiagnosticLogging.enable(_logger)
+            AzureStatusLogger.log_status(False, "Configurator being configured.")
             super()._configure(**kwargs)
+            AzureStatusLogger.log_status(True)
```

Comment:
I think we can remove the "being configured" log and simply just have `AzureStatusLogger.log_status(True, "AzureMonitorConfigurator successfully configured.")

Reply:
Sounds good.

Reply:
Done.

Reply:
Looks like it's still there? I'm referring to `AzureStatusLogger.log_status(False, "Configurator being configured.")`

Reply:
It's not there for me...